### DISCUSSION TITLE
Traduire les descriptions des commandes en fonction de la langue de discord

### DISCUSSION THREAD

**Discussion by Arkitu** [Crownicles/Crownicles#1351] [Idées acceptées / déjà implémentées]
C'est possible de traduire les noms et descriptions des commandes en fonction de la langue du client. Ça serait sympa de l'intégrer au bot (ça a pas l'air compliqué en plus). Peut être que laisser les noms des commandes serait bien mais les descriptions seraient mieux traduites.
https://discord.com/developers/docs/interactions/application-commands#localization

**Comment by BastLast** [comment]
on est déjà au courant et ç'est déjà prévu ya peut être meme déjà une issue. Si c'est pas le cas, go en créer une right now svp
